irohad/model/permissions.hpp

```diff
@@ -24,56 +24,57 @@
 namespace iroha {
   namespace model {
 
-    const std::string can_append_role = "CanAppendRole";
-    const std::string can_create_role = "CanCreateRole";
-    const std::string can_detach_role = "CanDetachRole";
-    const std::string can_add_asset_qty = "CanAddAssetQuantity";
-    const std::string can_subtract_asset_qty = "CanSubtractAssetQuantity";
-    const std::string can_add_peer = "CanAddPeer";
-    const std::string can_add_signatory = "CanAddSignatory";
-    const std::string can_create_account = "CanCreateAccount";
-    const std::string can_create_asset = "CanCreateAsset";
-    const std::string can_create_domain = "CanCreateDomain";
-    const std::string can_remove_signatory = "CanRemoveSignatory";
-    const std::string can_set_quorum = "CanSetQuorum";
-    const std::string can_transfer = "CanTransfer";
-    const std::string can_receive = "CanReceive";
-    const std::string can_set_detail = "CanSetAccountInfo";
+    //TODO 11/01/18 motxx - Use protobuf expressions or make consistent with shared model and old fashioned model permission identifiers.
+    const std::string can_append_role = "can_append_role";
```

Comment:
Why renamed?

Reply:
At first, I tried to send transaction for test this query.
A transaction in genesis block has `createRole()` (shared model), which leads error.
But I'll remove this test finally.

Reply:
Ok then